src/components/missions/MissionEntry.js

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types';
+import Button from 'react-bootstrap/Button';
+import Badge from 'react-bootstrap/Badge';
+
+const joinMessage = (status) => {
+  if (status === 'Not a member') {
+    return (
+      <Button
+        className="text-nowrap mx-3"
+        variant="outline-secondary"
+      >
+        Join Mission
+      </Button>
+    );
+  }
```

Comment:
[OPTIONAL]: here you can use the Conditional (ternary) operator for ES6 for best practice and to reduce your code : 
```JS
function joinMessage(status) {
  return (status === 'Not a member' && <Button
        className="text-nowrap mx-3"
        variant="outline-secondary"
      >
        Join Mission
      </Button>);
}
```
and after that use it inside your JSX code

Reply:
I didn't like that piece of code either. I'll find a better way to do it.